app/javascript/components/cloud-volume-form/index.jsx

```diff
@@ -20,7 +20,7 @@ const CloudVolumeForm = ({ recordId, storageManagerId }) => {
   useEffect(() => {
     if (recordId) {
       API.get(`/api/cloud_volumes/${recordId}`).then((initialValues) => {
-        API.options(`/api/cloud_volumes?ems_id=${initialValues.ems_id}`).then(loadSchema({ initialValues, isLoading: false }));
+        API.options(`/api/cloud_volumes?ems_id=${initialValues.ems_id}&record_id=${recordId}`).then(loadSchema({ initialValues, isLoading: false }));
```

Comment:
Should this actually be 
```suggestion
        API.options(`/api/cloud_volumes/${recordId}?ems_id=${initialValues.ems_id}`).then(loadSchema({ initialValues, isLoading: false }));
```

Incidentally, I think this is what you were looking for @GilbertCherrie

Reply:
@Fryguy I get the following error with the changes you suggested:
No route matches [OPTIONS] "/api/cloud_volumes/67"

I cant find were to set the routes for OPTIONS, can you please direct me?

Reply:
Oh that may be accurate - I thought OPTIONS on a single record should have worked, but it's apparently not available.

Reply:
@Fryguy any update?

Reply:
I think we can go this direction for now, but we'll probably want to switch it once we get OPTIONS on a specific record.  cc @GilbertCherrie @agrare

Reply:
Does the same apply for cloud_networks, as in should I just keep it as /api/cloud_networks?ems_id=id for now?